Doc 530 azure byo vnet phase 1 (#121)
redpanda-data/cloud-docs: modules/get-started/pages/cluster-types/byoc/azure/vnet-azure.adoc

  "redpanda_management_storage_container_name": "",
  "redpanda_tiered_storage_account_name": "",
  "redpanda_tiered_storage_container_name": "",
  "redpanda_management_key_vault_name": "",
  "redpanda_console_key_vault_name": "",
  "vnet_name": "",
  "vnet_addresses": "10.0.0.0/20",
  "private_subnets": {},
  "egress_subnets": {},
  "reserved_subnet_cidrs": {},
  "redpanda_security_group_name": ""
}
```

The https://github.com/redpanda-data/cloud-examples/blob/main/customer-managed/azure/terraform/README.md[Terraform Readme] has more information about the variable files: 

* `vars.azure.tf`: Contains the variables related to Azure credentials.
* `vars.condition.tf`: Contains the conditional variables; for example, on whether to create NAT.
* `vars.customer_input.tf`: Contains the variables needed to create a cluster with the Cloud API.
* `vars.iam.tf`: Contains the variables related to IAM roles.
* `vars.misc.tf`: Contains the variables, regions, zones, tags, and resource name prefix.

=== Initialize Terraform

Initialize the working directory containing Terraform configuration files:

```
terraform init
```

=== Apply Terraform

```
terraform apply
```

> The output of `terraform apply` displays output values. Note these values, because they are needed for the next steps. To continue with cluster creation, see https://github.com/redpanda-data/cloud-examples/tree/main/customer-managed/azure#create-azure-byo-vnet-redpanda-cluster[Create Azure VNet Redpanda Cluster^].

Reviewer: ```suggestion
The output of `terraform apply` displays output values. Note these values, because they are needed in subsequent steps. To continue with cluster creation, see https://github.com/redpanda-data/cloud-examples/tree/main/customer-managed/azure#create-azure-byo-vnet-redpanda-cluster[Create Azure VNet Redpanda Cluster^].
```
